apps/web/pages/auth/setup/steps/StepDone.tsx

```diff
@@ -1,20 +1,26 @@
 import { useRouter } from "next/router";
+import { Dispatch, SetStateAction } from "react";
 
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { FiCheck } from "@calcom/ui/components/icon";
 
-const StepDone = () => {
+const StepDone = (props: {
+  currentStep: number;
+  nextStepPath: string;
+  setIsLoading: Dispatch<SetStateAction<boolean>>;
```

Comment:
StepDone could be shown at any completed step so it was parametrized to accommodate any step.